runbotics-orchestrator/src/main/java/com/runbotics/service/dto/ProcessDTO.java

```diff
@@ -42,13 +42,7 @@ public class ProcessDTO implements Serializable {
     private ZonedDateTime updated;
 
     @JsonView(ProcessDTOViews.DefaultView.class)
-    private Long executionsCount;
-
-    @JsonView(ProcessDTOViews.DefaultView.class)
-    private Long successExecutionsCount;
-
-    @JsonView(ProcessDTOViews.DefaultView.class)
-    private Long failureExecutionsCount;
+    private ZonedDateTime lastRun;
```

Comment:
the time in the database shouldn't be without the timezone and adjusted on the frontend?

Reply:
Just like we discussed, I will leave it how it is.